projects/demo/src/modules/customization/routable/eager/routable-dialog.component.ts

```diff
@@ -16,7 +20,35 @@ export class RoutableDialogComponent {
         ),
     };
 
+    readonly example2: TuiDocExample = {
+        'page.template.html': import('./examples/2/page-2.component.html?raw'),
+        'page.module.ts': import('./examples/2/page-2.module.ts?raw'),
+        'dialog-content.component.ts': import(
+            './examples/2/dialog-content.component.ts?raw'
+        ),
+    };
+
     readonly addRouterOutlet = import('./examples/setup/add-router-outlet.md?raw');
     readonly importModule = import('./examples/setup/import-module.md?raw');
     readonly useRouteGenerator = import('./examples/setup/use-route-generator.md?raw');
+
+    /**
+     * Helps to destroy router-outlet for named outlet example
+     */
+    readonly isNamedOutletPage$ = this.router.events.pipe(
+        filter(isNavigationEndEvent),
```

Comment:
`filter(event => event instanceof NavigationEnd)` why just not?

Reply:
Got problem with typings. `filter` method just pass previous type(`Event`) as result after filtration.

<img width="846" alt="image" src="https://user-images.githubusercontent.com/38430641/223688480-f4fecf99-e8ac-48b8-a177-d340ad701475.png">